### DISCUSSION TITLE
Unable to get eglot to send `:snippetSupport t`, `eglot--snippet-expansion-fn` returns nil

### DISCUSSION THREAD

**Discussion by kcbanner** [joaotavora/eglot#1199] [General]
If I'm in a buffer with eglot enabled (and yas-minor-mode is enabled), if I eval `(eglot--snippet-expansion-fn)`, it returns nil.

However, if I eval its body, ie:

```
(and (fboundp 'yas-minor-mode)
       (lambda (&rest args)
         (with-no-warnings
           (unless (bound-and-true-p yas-minor-mode) (yas-minor-mode 1))
           (apply #'yas-expand-snippet args))))
```

Then it returns a result. Is this a misconfguration on my end? I ensured that the yas minor mode is before the hook that runs eglot.          
      
I'm on latest emacs master.

**Comment by kcbanner** [comment]
I had a `(advice-add 'company-tng--supress-post-completion :override #'ignore)` line that was overriding this function.

**Comment by joaotavora** [comment]
Alright. Be advised that the logic for snippet support announcement had changed very recently. The next release of Eglot won't need `yas-minor-mode`  to be explicitly enabled.
